hermes-api/src/main/java/pl/allegro/tech/hermes/api/Subscription.java

```diff
@@ -88,6 +88,8 @@
     @Valid
     private SubscriptionOAuthPolicy oAuthPolicy;
 
+    private boolean subscriptionIdentityHeadersEnabled;
```

Comment:
Should we duplicate domain(`subscription*`) for this field?
WDYT about `identityHeadersEnabled`?

Reply:
I think that in this case it is better to add `subscription` prefix because it makes it cleaner what kind of identity we are going to attach to message.